### DISCUSSION TITLE
Borg warnings about deprecated options

### DISCUSSION THREAD

**Discussion by p3k** [KenKundert/emborg#76] [Q&A] [answered]
I am seeing the following warnings when running emborg:

```shell
$ emborg version
emborg version: 1.37 (2023-05-18) [Python 3.9.2].
$ emborg log
…
Borg stderr:
    Warning: "--remote-ratelimit" has been deprecated. Use --upload-ratelimit instead.
    Warning: "--prefix" has been deprecated. Use "--glob-archives 'yourprefix*'" (-a) instead.
```

Is this something I can fix in the emborg config or does it need to be fixed in the emborg code?

**Comment by KenKundert** [comment, marked as answer]
I recommend that you read the documentation for each of the following settings:

[remote_ratelimit](https://emborg.readthedocs.io/en/stable/configuring.html#remote-ratelimit)
[upload_ratelimit](https://emborg.readthedocs.io/en/stable/configuring.html#upload-ratelimit)

[prefix](https://emborg.readthedocs.io/en/stable/configuring.html#prefix)
[glob_archives](https://emborg.readthedocs.io/en/stable/configuring.html#glob-archives)

Basically you can simply rename *remote_ratelimit* to *upload_ratelimit*.  Converting from *prefix* to *glob_archives* is similar, but requires a bit of massaging.

**Reply by p3k** [reply]
Oh OK! Think I better RTFM 😅 

Thanks for the pointers – after renaming or modifying the settings according to your suggestions and the documentation the warnings have disappeared.
